rusoto/services/cur/src/generated.rs

```diff
@@ -14,59 +14,51 @@ use rusoto_core::signature::SignedRequest;
 use serde_json::Value as SerdeJsonValue;
 use serde_json::from_str;
 #[doc="Region of customer S3 bucket."]
-pub type AWSRegion = String;
 #[doc="Enable support for Redshift and/or QuickSight."]
```

Comment:
Ahh, good catch. I think we can just put an `if shape.shape_type == ShapeType::Structure` around the doc generation as well to resolve this. Will fix that later today. :)